origin.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Andre Lau Portfolio</title>
+    <link rel="stylesheet" href="./main.css">
+</head>
+
+<body>
+    <header>
+        <nav class="navbar">
+            <h3>Hector Andre Lau</h3>
+            <a href=""><img src="./images/Union.svg" alt="Menu"></a>
+        </nav>
+    </header>    
+        
+        <h1>Hey there, I'm Hector Andre Lau</h1>
+        <h2>I'm a software developer with experience in finance and logistic processes.</h2>
+        <p>I can build a website meeting the expected requirements, pluss I can add my actual knowledge of how the process need to work on certain areas like logistic, account, finance and billing and all of this focus to the good experience and satisfaction ot the customer.</p>
+        <section>
+            <h2>Let's get in touch...!</h2>
+            <p>
+                <a href=""><img src="./images/facebooklogo.jpg" alt="facebooklogo"></a>
+                <a href=""><img src="./images/instagramlogo.jpg" alt="instagramlogo"></a>
+                <a href=""><img src="./images/linkedinlogo.jpg" alt="linkedinlogo"></a>
+                <a href=""><img src="./images/twitterlogo.png" alt="twitterlogo"></a>
+                <a href=""><img src="./images/githublogo" alt="githublogo"></a>
+            </p>
```

Comment:
- [ ] Please, consider, as requested by the previous reviewer, to put these social network icons into `li`. It means you have to implement `ul`/`li` structure, like the following ( I will show you a simple for your code, you will continue and fix all)

```
<ul>
     <li>
         <a href=""><img src="./images/facebooklogo.jpg" alt="facebooklogo"></a>
     </li>
     <li>
        <a href=""><img src="./images/instagramlogo.jpg" alt="instagramlogo"></a>
     </li>
</ul>
```